app/models/domain_block.rb

```diff
@@ -29,4 +29,11 @@ class DomainBlock < ApplicationRecord
   def self.blocked?(domain)
     where(domain: domain, severity: :suspend).exists?
   end
+
+  def stricter_than?(other_block)
```

Comment:
You could overload the `>` method here and that would be fun, but probably best not to